packages/core/src/external/opensearch/file/file-searcher-direct.ts

```diff
@@ -19,12 +20,14 @@ export class OpenSearchFileSearcherDirect implements OpenSearchFileSearcher {
 
     debug(`Searching on index ${indexName}...`);
     const queryPayload = {
+      size: 1_000,
       query: {
         bool: {
           must: [
             {
-              query_string: {
-                query,
+              // https://docs.opensearch.org/docs/latest/query-dsl/full-text/simple-query-string/
+              simple_query_string: {
```

Comment:
I should also add the conditional logic I added on `packages/core/src/external/opensearch/lexical/lexical-search.ts` so we can return all entries for this pt if the search is empty - allows to load everything.